server/handlers/default.go

```diff
@@ -129,7 +129,7 @@ func getHandler(ctx context.Context, w http.ResponseWriter, r *http.Request, var
 	out, err := store.GetCurrent(gun, tufRole)
 	if err != nil {
 		if _, ok := err.(storage.ErrNotFound); ok {
-			logrus.Error("404 GET " + gun + ":" + tufRole)
+			logrus.Error("404 GET " + gun + ":" + tufRole + ", error:" + err.Error())
```

Comment:
You did the right thing preserving the existing code format, but while you're here, could you switch this over to `logrus.Errorf("404 GET %s:%s, error: %v", gun, tufRole, err)`